channel-jira-server/src/test/java/com/synopsys/integration/alert/channel/jira/server/action/JiraServerGlobalCrudActionsTest.java

```diff
@@ -0,0 +1,171 @@
+import com.synopsys.integration.alert.channel.jira.server.validator.JiraServerGlobalConfigurationValidator;
+import com.synopsys.integration.alert.common.action.ActionResponse;
+import com.synopsys.integration.alert.common.enumeration.ConfigContextEnum;
+import com.synopsys.integration.alert.common.persistence.model.PermissionKey;
+import com.synopsys.integration.alert.common.persistence.model.PermissionMatrixModel;
+import com.synopsys.integration.alert.common.rest.AlertRestConstants;
+import com.synopsys.integration.alert.common.rest.model.AlertPagedModel;
+import com.synopsys.integration.alert.common.security.authorization.AuthorizationManager;
+import com.synopsys.integration.alert.common.util.DateUtils;
+import com.synopsys.integration.alert.descriptor.api.JiraServerChannelKey;
+import com.synopsys.integration.alert.descriptor.api.model.DescriptorKey;
+import com.synopsys.integration.alert.test.common.AuthenticationTestUtils;
+
+public class JiraServerGlobalCrudActionsTest {
+    private final String CREATED_AT = String.valueOf(DateUtils.createCurrentDateTimestamp().minusMinutes(5L));
+    private final String UPDATED_AT = String.valueOf(DateUtils.createCurrentDateTimestamp());
+    private final String URL = "https://someUrl";
+    private final String USER_NAME = "username";
+    private final String PASSWORD = "password";
+
+    private final UUID id = UUID.randomUUID();
+
+    private final AuthenticationTestUtils authenticationTestUtils = new AuthenticationTestUtils();
+    private final DescriptorKey descriptorKey = new JiraServerChannelKey();
+    private final PermissionKey permissionKey = new PermissionKey(ConfigContextEnum.GLOBAL.name(), descriptorKey.getUniversalKey());
+    private final Map<PermissionKey, Integer> permissions = Map.of(permissionKey, 255);
+    private final AuthorizationManager authorizationManager = authenticationTestUtils.createAuthorizationManagerWithCurrentUserSet("admin", "admin", () -> new PermissionMatrixModel(permissions));
+
+    private final JiraServerGlobalConfigurationValidator validator = new JiraServerGlobalConfigurationValidator();
+
+    @Test
+    void getOneTest() {
+        JiraServerGlobalConfigModel jiraServerGlobalConfigModel = createJiraServerGlobalConfigModel(id);
+
+        JiraServerGlobalConfigAccessor configAccessor = Mockito.mock(JiraServerGlobalConfigAccessor.class);
+        Mockito.when(configAccessor.getConfiguration(id)).thenReturn(Optional.of(jiraServerGlobalConfigModel));
+
+        JiraServerGlobalCrudActions crudActions = new JiraServerGlobalCrudActions(authorizationManager, configAccessor, validator);
+        ActionResponse<JiraServerGlobalConfigModel> actionResponse = crudActions.getOne(id);
+
+        assertTrue(actionResponse.isSuccessful());
+        assertTrue(actionResponse.hasContent());
+        assertEquals(HttpStatus.OK, actionResponse.getHttpStatus());
+        assertModelObfuscated(actionResponse);
+    }
+
+    @Test
+    void getOneEmptyTest() {
+
```

Comment:
Not sure if you're aware of this specific empty test

Reply:
My bad, meant to delete that one, will be covered in the IT test